verification/rotateContextHandle.go

```diff
@@ -0,0 +1,78 @@
+// Licensed under the Apache-2.0 license
+
+package verification
+
+import (
+	"errors"
+	"testing"
+)
+
+func TestRotateContextHandle(d TestDPEInstance, c DPEClient, t *testing.T) {
+	testRotateContextHandle(d, c, t, false)
+}
+
+func TestRotateContextHandleSimulation(d TestDPEInstance, c DPEClient, t *testing.T) {
+	testRotateContextHandle(d, c, t, true)
+}
+
+func testRotateContextHandle(d TestDPEInstance, c DPEClient, t *testing.T, simulation bool) {
+	handle := getInitialContextHandle(d, c, t, simulation)
```

Comment:
Tests the RotateContext command with command flag.
The spec provides only one command flag TARGET_IS_DEFAULT.
Have checked used/unused command flag in default/simulation mode

Reply:
Hi @jhand2 
Kindly take a look at this PR containing tests for ROtateContext() command
This may be considered for review once PRs 253 and 251 are merged to overlook changes pertaining to ExtendTCi and  Sign command.